src/content/docs/dropins/user-auth/slots.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: User auth slots
+description: Learn about the slots provided in the UserAuth drop-in component.
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+
+The user auth drop-in component provides several slots that can be used to customize the presentation of its containers.
+
+[Extending drop-in components](/dropins/all/extending/) describes default properties available to all slots.
+
+## PrivacyPolicyConsent slot
+
+The `PrivacyPolicyConsent` slot can be used to display the privacy policy consent checkbox in the [`SignUp` container](/dropins/user-auth/containers/sign-up/). The following example shows how to use this slot to display a custom privacy policy message and link, as shown in the image below.
```

Comment:
Not a checkbox, just a static privacy consent message (see screenshot added in this PR).